releases/client/settings

```diff
@@ -0,0 +1,2 @@
+OSES="rhel5 rhel6 rhel7 sles11 sles12 fedora28 fedora29"
```

Comment:
Should we explicitly copy this so we don't have to when we change 1.24 (or later) to drop F28?

Reply:
you mean to `releases/client/1.23/settings`? probably yes.

Reply:
Exactly.